bootstrap.sh

```diff
@@ -1,54 +1,67 @@
-#!/bin/bash
+#!/usr/bin/bash
 
-set -eux pipefail
+set -e  # Exit with nonzero exit code if anything fails
 
 echo "Validate sudo credentials ..."
 sudo -v
 
 
-#Add Deb Servers - Spotify
+#Add Deb Server - Spotify
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 931FF8E79F0876134EDDBDCCA87FF9DF48BF1C90
 echo deb http://repository.spotify.com stable non-free | sudo tee /etc/apt/sources.list.d/spotify.list
 
-#Add Deb Servers - Atom
+#Add Deb Server - Atom
 wget -qO - https://packagecloud.io/AtomEditor/atom/gpgkey | sudo apt-key add -
 sudo sh -c 'echo "deb [arch=amd64] https://packagecloud.io/AtomEditor/atom/any/ any main" > /etc/apt/sources.list.d/atom.list'
 
+#Add Deb Server - Chrome
+wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+echo 'deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main' | sudo tee /etc/apt/sources.list.d/google-chrome.list
 
-#Updating Package List
+
+#Updating Package List and upgrade
 sudo apt-get update
+sudo apt-get dist-upgrade
+
+#Software Install - System Utilities
+sudo apt-get install -y htop tree
+
+#Software Install - Chrome
+sudo apt-get install google-chrome-stable
 
 #Software Install - Blue Light Dimmer
 sudo apt-get install -y redshift
-
 #Configure Redshift
-#add time zone and birtness @(TODO)
-#add autostart
-
-#Software Install - System Utilities
-sudo apt-get install -y htop tree
+#TODO: config redshift for birtness and autostart
 
 #Software Install - Spotify
 sudo apt-get install -y spotify-client
-
 #Cofigure Spotify
-#sign-on and spotify cli @(TODO)
+#TODO: sign-on and spotify cli
 
 #Software Install - Atom
 sudo apt-get install -y atom
-
-#Configure Atom 
-apm install autocomplete-python
+#Configure Atom - Package Install
 apm install highlight-selected
+apm install autocomplete-python
+apm install atom-beautify
 apm install language-docker
-apm install language-dotenv
 apm install linter-kubectl
-#theme install and settings cofigure @(TODO)
+apm install markdown-preview
+apm install todo-show
+apm install open-in-browser
+apm install highlight-selected
+#Configure Atom - Theme Install
```

Comment:
Look at this [guide](https://code.visualstudio.com/docs/setup/linux) on how to do this! So dope!

Reply:
atom for now but will try VS code